swaybar/tray/icon.c

```diff
@@ -89,6 +89,9 @@ static bool validate_icon_theme(struct icon_theme *theme) {
 static bool group_handler(char *old_group, char *new_group,
 		struct icon_theme *theme) {
 	if (!old_group) { // first group must be "Icon Theme"
+		if (!new_group) {
+			return -1;
```

Comment:
`group_handler` should return a `bool`

Reply:
True, but `strcmp` doesn't either. `strcmp("", "something")` returns `-1`.

Reply:
I return `-1` mainly to be consistent with the other case. I could just return `true` if you prefer that.

Reply:
Yeah, that'd be better. Changing the `strcmp` call to return `strcmp(…) != 0` would be nice too.

Reply:
Done :+1: